.github/workflows/release.yml

```diff
@@ -0,0 +1,37 @@
+name: Core
+
+on:
+  push:
+    tags: ['release']
```

Comment:
Will this work? We are not tagging commits with tag release but with version tags.